kmesdk/src/main/java/com/kme/kaltura/kmesdk/content/poll/KmeDefaultPollEventHandler.kt

```diff
@@ -14,17 +16,14 @@ class KmeDefaultPollEventHandler(
     private val roomController: IKmeRoomController
 ) {
 
-    private val pollStarted = MutableLiveData<QuickPollStartedPayload>()
-    val pollStartedLiveData
-        get() = pollStarted as LiveData<QuickPollStartedPayload>
+    private val pollStarted = LiveEvent<QuickPollStartedPayload>()
+    val pollStartedLiveData get() = pollStarted.toSingleEvent()
```

Comment:
As I remember .toSingleEvent() breaks single event logic. PLease check this on room login/logout and remove if so.

Reply:
I mean should be like this (if breaks):

private val pollStarted = LiveEvent<QuickPollStartedPayload>()
val pollStartedLiveData get() = pollStarted

Reply:
Done